test-case/manual/check-pulseaudio.md

```diff
@@ -0,0 +1,61 @@
+# Pulseaudio test
+Pulseaudio testing is manual testing for now.
+
+# Preconditions
+1. PulseAudio should be enabled
+2. UCM for the platform/device should be installed
+
+# Test Description
+* Change current audio device throught Setting->Sound app
```

Comment:
may we say the test is designed for Ubuntu?
```suggestion
* Change current audio device through Setting->Sound app
```

Reply:
I can change to "Setting->Sound app in Ubuntu"